bundle.yml

```diff
@@ -54,14 +54,19 @@ projects :
             ECKIT_ENABLE_ECKIT_SQL=OFF
             ECKIT_ENABLE_WARNINGS=OFF
 
-    - field_api :
-        git     : https://github.com/ecmwf-ifs/field_api
-        version : develop-ecmwf
+    - fiat      :
```

Comment:
As discussed, fiat should be optional to keep the dwarf itself as minimal as possible, which has proven useful in the past. Thus, add a `BUILD_fiat=OFF` in l. 13 to disable fiat by default.